observations/obs_converters/README.rst

```diff
@@ -421,11 +413,20 @@ For generating synthetic observations, see the
 `create_obs_sequence <../../assimilation_code/programs/create_obs_sequence/create_obs_sequence.html>`__
 program documentation. You can also generate observation files based on
 text input. See the `text_to_obs <text/text_to_obs.html>`__ program
-documentation. Or for simulating a large complex observing system, you
+documentation and even_sphere. Or for simulating a large complex observing system, you
 can use the DART library routines in a Fortran program to compute the
 observation information and have the DART routines write the output
 file.
 
+There are a couple utilities of note:
+
+-  `even_sphere <even_sphere/README.html>`__ - a utility for generating
+   a text file of evenly-spaced observation locations that can then be used in a
+   perfect model experiment.
```

Comment:
this is good.  we don't mention this utility anywhere and it could be useful for testing global models like cam and mpas.   maybe clarify two things - that it makes 3d sphere global obs (so not useful for oned models), and that the output isn't text in the sense of what text_to_obs would read, but specifically input to create_obs_sequence, something like:

```suggestion
-  `even_sphere <even_sphere/README.html>`__ - a utility for generating
   a text file of evenly-spaced observation locations around a globe 
   that can then be used by `create_obs_sequence <../../assimilation_code/programs/create_obs_sequence/create_obs_sequence.html>`__ to make 
   an input observation sequence for a perfect model experiment.
```

disclaimer - i don't know if i've formatted the reference at all right.  i'm just copying other references.